docs/source/daml/reference/interfaces.rst

```diff
@@ -41,9 +44,10 @@ Interface Methods
    :end-before: -- INTERFACE_METHODS_END
 
 - An interface may define any number of methods.
-- Methods are in scope as functions at the top level, in the ensure clause, and
-  in interface choices. These functions always take an unstated first argument
-  corresponding to a contract that implements the interface:
+- Methods are in scope as functions at the top level and in interface choices.
+  These functions always take an unstated first argument corresponding to a
+  contract of a template type that is an ``interface instance`` of the
```

Comment:
it’s not a contract, it’s the interface value/argument/payload.

Reply:
and slightly confusingly in the definition it’s the type of the template, when you call it it’s the type of the interface. probably worth pointing tha tout somewhere.

Reply:
I'll split this into two bullet points